src/fuse_api.pxi

```diff
@@ -387,7 +387,7 @@ cdef void* worker_start(void* data) with gil:
         session_loop(wd.buf, wd.bufsize)
     except:
         fuse_session_exit(session)
-        tid = wd.thread_id
+        tid = <uintptr_t> wd.thread_id
```

Comment:
For the record, I don't think there's a guarantee that `pthread_t` will be cleanly castable to an integer but I suppose there's no harm doing that until someone actually tries to use with with non-Linux FUSE implementation and complains ;-).

Reply:
Is there a better way to implement this and avoid the compiler warning? tid is only used for log output.

Reply:
I'm afraid this is a non-trivial problem. Roughly, you could:

1. "dump" `pthread_t` but it's probably not worth the complexity for a debug print (basically print it as hex byte-by-byte)
2. use non-portable getter inside the thread to get its numeric TID and pass it but it's complex and then you'd need extra compatibility if different platforms are involved, see e.g. https://github.com/llvm/llvm-project/blob/llvmorg-16.0.3/llvm/lib/Support/Unix/Threading.inc#L113-L134
3. just create your own thread numbers and assign them consecutively (e.g. `thread 1`, `thread 2` and so on)

See e.g. https://stackoverflow.com/questions/1759794/how-to-print-pthread-t

Reply:
thanks for the info, I created https://github.com/python-llfuse/python-llfuse/issues/85 so we can improve this later.